orchestrate.py

```diff
@@ -327,7 +332,9 @@ def main():
 
     handlers = [
         (r"/api/spawn/?", APISpawnHandler),
-        (r"/api/stats/?", APIStatsHandler)
+        (r"/api/stats/?", APIStatsHandler),
+        (r"/stats/?", RedirectHandler, {"url": "/api/stats"}),
+        (r"/info/?", InfoHandler),
```

Comment:
In general, Using the tornado get_current_user is way better than what I did. Totally forgot it existed. Nice!

Adding the unprotected `/info` handler back in here is a problem in an environment where tmpnb needs to be accessible only programmatically and only by folks with the API token. But I understand why you might want it as an admin.

Maybe tmpnb should have another flag about enabling / disabling it separately from the API token?